trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/generator/RtInconcistencyConditionGenerator.java

```diff
@@ -307,7 +307,8 @@ private static Script buildSolver(final IUltimateServiceProvider services) throw
 	 *
 	 */
 	public Expression generateNonDeadlockCondition(final PhaseEventAutomata[] automata) {
-		if (PRINT_PEA_DOT) {
+		//if (PRINT_PEA_DOT) {
+		if (true) {
```

Comment:
Please do not remove the printing switch:

```suggestion
		if (PRINT_PEA_DOT) {
```